index.html

```diff
@@ -0,0 +1,557 @@
+              alt=""
+            />
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-twitter.png"
+              alt=""
+            />
+          </div>
+        </div>
+
+        <div class="card-team">
+          <img class="card-team__img" src="./assets/images/media.png" alt="" />
+
+          <p class="card-team__title">Ashley Fletcher</p>
+          <p class="card-team__subtitle">ceo</p>
+
+          <div class="social">
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-facebook.png"
+              alt=""
+            />
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-instagram.png"
+              alt=""
+            />
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-twitter.png"
+              alt=""
+            />
+          </div>
+        </div>
+
+        <div class="card-team">
+          <img
+            class="card-team__img"
+            src="./assets/images/cover-2.png"
+            alt=""
+          />
+
+          <p class="card-team__title">Ashley Fletcher</p>
+          <p class="card-team__subtitle">ceo</p>
+
+          <div class="social">
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-facebook.png"
+              alt=""
+            />
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-instagram.png"
+              alt=""
+            />
+            <img
+              class="social__icon"
+              src="./assets/images/Vector-twitter.png"
+              alt=""
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="appointment">
+      <div class="contents">
+        <h3 class="contents__title">Contact us</h3>
+        <p class="contents__description">Make an appointment</p>
+
+        <div class="input">
+          <input class="input__item" type="text" placeholder="Full Name" />
+          <input
+            class="input__item"
+            type="text"
+            placeholder="example@gmail.com"
+          />
+          <input class="input__item" type="text" placeholder="Please Select" />
+          <input class="input__item" type="text" placeholder="4:00 Available" />
+        </div>
+        <textarea class="contents___area" name="" id="" cols="30" rows="10">
+        </textarea>
+        <button class="contents__btn">Book Appointment</button>
+      </div>
+    </section>
+
+    <section class="last">
+      <div class="last__content">
+        <h1 class="last__title">Consulting Agency For Your Business</h1>
+        <p class="last__description">the quick fox jumps over the lazy dog</p>
+      </div>
+
+      <button class="last__btn">Contact Us</button>
+    </section>
+
+    <!-- Footer -->
+    <section class="footer-container">
```

Comment:
improve this naming of `footer-container`. may be name it `footer`. and the next inline block name it `footer_container`